scripts/compute_step7_kpis.py

```
@@ -0,0 +1,127 @@
+    baseline_x1 = 0.8317  # from wkmigrate-issue-27-meta-kpis.md
+
+    # X-6: per-canonical-category resolution
+    vc_cat = _per_category(vc_if)
+    crp_cat = _per_category(crp_if)
+
+    # Secondary: VC all-contexts
+    vc_all_ctx = vc_all.get("by_context", {})
+
+    out = []
+    out.append("# Step 7 — Vista Cliente Sweep KPI Table\n")
+    out.append(f"Generated: 2026-04-23  |  wkmigrate SHA: cfb49e6\n\n")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)